org-code-javabuilder/lib/src/main/java/org/code/javabuilder/BaseTestRunner.java

```diff
@@ -22,16 +23,19 @@
   private final JavabuilderTestExecutionListener listener;
   private final OutputAdapter outputAdapter;
   private final StatusMessageKey statusMessageKey;
+  private final boolean isValidation;
 
   public BaseTestRunner(
       List<JavaProjectFile> files,
       JavabuilderTestExecutionListener listener,
       OutputAdapter outputAdapter,
-      StatusMessageKey statusMessageKey) {
+      StatusMessageKey statusMessageKey,
```

Comment:
Could we just use the value of `statusMessageKey` to tell if this is validation or not?

Reply:
good idea. I think I might actually remove `statusMessageKey` here and use `isValidation` to pick the key.